CSharpFunctionalExtensions/CSharpFunctionalExtensions.csproj

```diff
@@ -47,4 +47,9 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)'!='net5.0' AND '$(TargetFramework)'!='net6.0'">
```

Comment:
can it be refactored to be more resilient?
For example the check will fail for .NET 7 and newer, so the serialization logic will blindly stop working as expected

Reply:
Fixed